README.md

```diff
@@ -1,6 +1,7 @@
 # Kotlinter Gradle
 
-[![Build Status](https://api.travis-ci.org/jeremymailen/kotlinter-gradle.svg?branch=master)](https://travis-ci.org/jeremymailen/kotlinter-gradle)
+[![Build Status](https://github.com/jeremymailen/kotlinter-gradle/workflows/Build%20Project/badge.svg)](https://github.com/jeremymailen/kotlinter-gradle/actions)
+[![Latest Version](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/org/jmailen/gradle/kotlinter-gradle/maven-metadata.xml?label=gradle)](https://plugins.gradle.org/plugin/org.jmailen.kotlinter)
```

Comment:
I updated old badge and added new one, with the latest published artifact version on Gradle Plugin Portal
<img width="352" alt="image" src="https://user-images.githubusercontent.com/36954793/158027422-ad2577f0-6e10-495b-96a2-ad928b8e670f.png"> 
(the "gradle" label can be set to whatever you want. Usually I saw there `jCenter` or `mavenCentral`, but `Gradle Plugin Portal` seemed too _lengthy_ :P)

The old badge was outdated, right? All CI pipelines have already been migrated to Github Actions, haven't they? (based on that assumption I removed the `.travis.yml` file

Reply:
Yes, thank you!